Formula/watch.rb

```diff
@@ -2,8 +2,8 @@ class Watch < Formula
   desc "Executes a program periodically, showing output fullscreen"
   homepage "https://gitlab.com/procps-ng/procps"
   url "https://gitlab.com/procps-ng/procps.git",
-      tag:      "v3.3.16",
-      revision: "59c88e18f29000ceaf7e5f98181b07be443cf12f"
+      tag:      "v3.3.17",
+      revision: "19a508ea121c0c4ac6d0224575a036de745eaaf8"
   license "GPL-2.0"
```

Comment:
```suggestion
  license "LGPL-2.1-or-later"
```
This seems to be what's in their `COPYING` file and source headers: https://gitlab.com/search?group_id=162935&nav_source=navbar&project_id=247135&repository_ref=master&scope=&search=at+your+option&search_code=true&snippets=false